grafana-formula/grafana/init.sls

```diff
@@ -1,5 +1,6 @@
 # check for supported os version
-{%- set supported_vers = ['42.3', '12.3', '12.4', '12.5', '15.0', '15.1', '15.2', '15.3', '15.4', '15.5', '15.6'] %}
+{%- set supported_vers = ['42.3', '12.3', '12.4', '12.5', '15.0', '15.1',
+                          '15.2', '15.3', '15.4', '15.5', '15.6', '15.7'] %}
```

Comment:
I'm wondering if, soon or later, we should also drop unsupported versions. 42.3 is gone for example. 
but I'm not 100% sure if that is a mix of Leap and SLE versions or what else

Reply:
Leap 42.3 is closely aligned with SLES 12 SP3. LTSS for the later has ended 3 years ago. On the other hand [metrics.opensuse.org](https://metrics.opensuse.org) shows around 3400 unique downloads for Leap 42.3 every month.
@admd What is your decision on this?

Reply:
> On the other hand [metrics.opensuse.org](https://metrics.opensuse.org) shows around 3400 unique downloads for Leap 42.3 every month. 

it's scary to see such  an old and unsupported distro still in  use :( 

but, if you are aligned with the SLE versions, 12.3 and 12.4 are both gone. the last supported 12.X still under maintenance is 12-SP5